msi/release.sh

```diff
@@ -170,15 +170,16 @@ echo "$LIST" |
 sort |
 uniq |
 tr / \\\\ |
-sed -e 's/\(.*\)\\\(.*\)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
+sed -r -e 's/(.*)\\(.*)/            <Component Directory="INSTALLFOLDER:\\\1\\">\
                 <File Source="&" \/>\
             <\/Component>/' \
-	-e 's/^\([^\\]*\)$/            <Component Directory="INSTALLFOLDER">\
-            <File Source="&" \/>\
-        <\/Component>/' \
-	-e 's/\(<File Source="git-bash.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git Bash" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/' \
-	-e 's/\(<File Source="git-cmd.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git CMD" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/' \
-	-e 's/\(<File Source="cmd\\git-gui.exe"[^>]*\) \/>/\1 \/><Shortcut Name="Git GUI" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/'
+	-e 's/^([^\\]*)$/            <Component Directory="INSTALLFOLDER">\
+                <File Source="&" \/>\
+            <\/Component>/' \
+	-e "s/<File Source=\"(mingw$BITNESS.*(ca)-(bundle\.crt))\" \/>/<File Id=\"\2_\3\" Source=\"\1\" \/>/"\
+	-e 's/(<File Source="git-bash.exe"[^>]*) \/>/\1 \/><Shortcut Name="Git Bash" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/' \
+	-e 's/(<File Source="git-cmd.exe"[^>]*) \/>/\1 \/><Shortcut Name="Git CMD" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/' \
+	-e 's/(<File Source="cmd\\git-gui.exe"[^>]*) \/>/\1 \/><Shortcut Name="Git GUI" Icon="git.ico" Directory="GitProgramMenuFolder" WorkingDirectory="INSTALLFOLDER" Arguments="--cd-to-home" Advertise="yes" \/>/'
```

Comment:
On Wed, 18 May 2016, Craig E. Shea wrote:

> You made a note on another commit where i used `-r` about it not being "as portable". Is the lack of the `-r` option for sed really a concern? (This is more to satisfy my curiosity than anything else.) I am under the impression that `-r` was fairly "portable" by now. I think that if this truly is not all that portable, then `-r` shouldn't be used (and then I'll just go cross-eyed :wink:).

AFAIR BSD's sed (and therefore MacOSX') do not support `-r`. Not that it
matters as far as Git for Windows is concerned. *I* am so used to look at
Git's source code, though, which *is* concerned with that.